.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
```

Comment:
Enter clicking on keyboard actually clicks previously clicked button on the app, it shouldn't be doing that, is should execute the operation the same as = on app and in keyboard, it results in strange behaviours currently